context/app/static/js/pages/Template/Template.tsx

```diff
@@ -141,6 +142,14 @@ function Template({ templateKey }: TemplatePageProps) {
               </Stack>
             </LabelledSectionText>
           )}
+          {template?.last_modified_unix_timestamp && (
+            <LabelledSectionText
+              label="Last Modified"
+              iconTooltipText="Date when this template was last modified by its template provider."
```

Comment:
We were just discussing it on the call but I missed the conclusion of this discussion - I feel like the second `template` here is a little redundant.

```suggestion
              iconTooltipText="Date when this template was last modified by its provider."
```

Reply:
@tsliaw thoughts?

Reply:
Okay I wasn't sure about that second template, I'm fine for removing it.